build_runner/lib/src/generate/build_impl.dart

```diff
@@ -221,18 +221,18 @@ class BuildImpl {
   Future<Set<AssetId>> _matchingInputs(InputSet inputSet, int phaseNumber,
```

Comment:
I wonder if we need a different name for this method...

Reply:
It does return a set of matching inputs still, but its true the name doesn't indicate that it is doing a lot of work potentially.